packages/@sanity/cli/src/util/neatStack.ts

```diff
@@ -5,26 +5,26 @@
  */
 import {inspect} from 'node:util'
 
-import {dim, red} from 'chalk'
+import chalk from 'chalk'
```

Comment:
Discovered that `import {dim, red} from 'chalk'` doesn't work if `@sanity/cli` has `"type": "module"`.
Even though I eventually moved back to using `"type": "commonjs"` for `@sanity/cli` I kept this change since it makes it easier in the future to migrate